tools/modules/update.sh

```diff
@@ -16,28 +16,30 @@
 # update_modules.sh updates the repositories for external code (to the precise
 # revisions used for testing).
 #
-# If the single argument `--git_only` is passed, the repositories will be
+# If the single argument `--fetch_only` is passed, the repositories will be
 # updated to the pinned versions without configuring or building them.
 # If the argument `--build_only` is passed, the script will assume that the
 # repositories are at the correct version and will configure and build them.
 
-git_maybe_clone() {
-  local repo="$1"
+wget_copy_archive() {
+  local target="$1"
   local dir="$2"
-  if [[ ! -d "$dir/.git" ]]; then
-    git clone "$repo" "$dir"
-  fi
-}
+  local sha="$3"
 
-git_checkout_sha() {
-  local repo="$1"
-  local sha="$2"
-  cd "$repo"
-  if [[ "$sha" != "$(git rev-parse HEAD)" ]]; then
-    git fetch origin master
-    git checkout -f "$sha"
+  if [[ ! -f "$dir/$sha.sentinel" ]]; then
+    wget "https://github.com/llvm-mirror/$target/archive/$sha.zip"
+    rm -rf "$dir"
+    mkdir -p "$dir"
+    local tmpdir=$(mktemp -d)
+    unzip "$sha.zip" -d "$tmpdir/"
+    # This relies on the behavior of github to always produce a zip archive with
+    # a subdirectory named "repo-###sha###" that contains the repo inside it.
+    mv -T "$tmpdir/$target-$sha" "$dir"
```

Comment:
`-T` is not a valid option on the OSX `mv` tool.

Reply:
Ah shit, what's the behavior in OSX mv to rename a directory then?

I'm not seeing one on https://www.unix.com/man-page/osx/1/mv/

:(

Reply:
`mv olddir newdir/newname`

Reply:
ah k yea because I'm removing the directory this should work.  I was leaving it in place with a previous revision (before settling on removing old versions).